packages/command-palette/src/commands/use-single-site-commands.tsx

```diff
@@ -528,15 +530,17 @@ const useSingleSiteCommands = ( { navigate, currentRoute }: useCommandsParams ):
 				),
 				_x( 'audit log', 'Keyword for the Open activity log command', __i18n_text_domain__ ),
 			].join( ' ' ),
-			callback: commandNavigation( '/activity-log/:site' ),
+			callback: commandNavigation(
+				`${ shouldUseWpAdmin ? 'https://cloud.jetpack.com' : '' }/activity-log/:site`
+			),
 			filterP2: true,
 			icon: acitvityLogIcon,
 		},
 		{
 			name: 'openJetpackBackup',
 			label: __( 'Open Jetpack Backup', __i18n_text_domain__ ),
 			callback: commandNavigation(
-				`${ siteType === SiteType.ATOMIC ? 'https://cloud.jetpack.com' : '' }/backup/:site`
+				`${ shouldUseWpAdmin ? 'https://cloud.jetpack.com' : '' }/backup/:site`
```

Comment:
This change redirects Simple sites using the Classic view to `cloud.jetpack.com`. Is this intended behavior?

Reply:
Currently, only Atomic sites can change the admin interface style to Classic (under Settings -> Hosting Configuration).

However, now that we want add that setting to simple sites as well (soon-ish), I just went ahead and updated the URLs to use Jetpack Cloud redirection instead. This will send Atomic sites to Jetpack Cloud, and simple sites to wordpress.com.

Reply:
We should probably leave this as is for now.

I don't know if simple sites are going to use Jetpack cloud or not something @arthur791004  will need to figure out in shaping pfsHM7-nh-p2

Reply:
Nevermind, just say the swap to JP rediect, that should work ok for simple yeah?

Reply:
Yup, for simple sites the url will still redirect to wp.com.